python/cuml/fil/README.md

```diff
@@ -0,0 +1,77 @@
+# FIL - RAPIDS Forest Inference Library
+
+The Forest Inference Library provides a lightweight, flexible API to
+infer (predict) results from a tree-based model ensemble on GPU. The
+tree ensemble can be either gradient-boosted decision tree (GBDT) or
+random forest (RF) models trained in XGBoost, cuML, scikit-learn, or
+LightGBM.
+
+# Code sample
+
+Starting with an XGBoost classification model saved in the file
+"xgb.mod," we want to use that model to infer on a large dataset of
+test samples.
+
+```python
+from cuml import ForestInference
+
+fm = ForestInference.load(filename=model_path,
+                          output_class=True,
+                          threshold=0.50,
+                          model_type='xgboost')
+
+X = ... load test samples as a numpy or cupy array ...
+
+y_out = fm.predict(X)
+
+```
+
+See [the sample notebook](https://github.com/rapidsai/cuml/blob/branch-0.16/notebooks/forest_inference_demo.ipynb) for much more detail and runnable samples.
```

Comment:
Is it possible to refer to the same branch as the one this file comes from, rather than hardcoding it?

Reply:
Added a reference to "main" which will have the stable release.